src/components/bark/login/layout.tsx

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image";
+import { BarkH1 } from "../bark-typography";
+import { BarkNavRoute } from "../bark-nav";
+import Link from "next/link";
+
+const LoginLayout = ({
+  accountType,
+  children,
+}: Readonly<{ accountType: string; children: React.ReactNode }>) => {
+  let logoSrc = "";
+
+  if (accountType === "ADMIN") {
+    logoSrc = "/purpleDogHouse.svg";
+  } else if (accountType === "VET") {
+    logoSrc = "/pawPrint.svg";
+  } else if (accountType === "USER") {
+    logoSrc = "/orangeDogHouse.svg";
+  }
```

Comment:
I reckon it would be cleaner to pass this in as a prop to BarkLoginPage, so that where ever BarkLoginPage is being used is where the AccountType differentiations is defined.

Reply:
Done d957f39b7761050266ba152c90f42e15602c7495